### ISSUE TITLE
Another Repo to Look at

### ISSUE THREAD

**Issue by ZeroShadowX** [CryptoNeverSleeps/DELL-XPS-7590-OLED#2]
I didn't want to comment in romancin's repo and talk about another repo (wasnt sure if that was proper) after I saw your message about you buying a 4K Screen. I wanted to let you know that a guy and I on reddit have been working on the our laptop and managed to get Big Sir working on it. Weve been actively working on it the last week and made a lot of progress, besides some small quirks we found yesterday everything works great. 

https://github.com/stakeout55/Dell-XPS-7590-mac-OS-Big-Sur-11.1

Hope that helps

**Comment by CryptoNeverSleeps** [comment]
I'm Actually using that repo... Thank You...

Problem is..... He has 4K IPS Touchscreen (The screen I'm thinking about buying for $400.00 USD)

OLED Screens will not wake from sleep... It's beyond my expertise to fix this.

**Comment by stakeout55** [comment]
Hey - are you guys discussing my repo? If so — My OLED wakes from sleep just fine!

**Comment by CryptoNeverSleeps** [comment]
Yes we are... I was looking at your repo and it says you don't have the OLED but the 4K IPS Touchscreen... This one has always worked...   OLED model is 4K No Touch Screen.

**Comment by stakeout55** [comment]
Sorry - yes I understand now. Hmmmm, am unsure on that. Are you considering an afterMarket 4K OLED touch?

**Comment by CryptoNeverSleeps** [comment]
No.  
Only models made are:

C41VT NEW OEM Dell XPS 7590 Precision 5540 UHD 3840x2160 IPS Touch Screen Grey 

https://ebay.us/opSaQw



4K OLED - has never worked when waking from sleep for anyone for with Dell Laptop.

4K IPS Touchscreen - Always worked. 10.15.7 - Big Sur - until whatever green 60Hz Fix....

1080P 
1080P Touchscreen?

**Comment by stakeout55** [comment]
I understand now. That screen should work. Never done any types of swap outs. Are you going to go through with it? And what do you have on there now? Super curious

**Comment by CryptoNeverSleeps** [comment]
My system is maxed out as far as specs ... I would love to keep the OLED... I'm at my limits.  Need help trying to figure it out...

I have 4K OLED.

**Comment by CryptoNeverSleeps** [comment]
According to your post you're using the screen I'm most likely purchasing this week on eBay.

**Comment by CryptoNeverSleeps** [comment]
![Uploading Screen Shot 2020-12-28 at 5.44.42 AM.png…]()

**Comment by CryptoNeverSleeps** [comment]
![Uploading AboutThisMac.png…]()

**Comment by CryptoNeverSleeps** [comment]
![About This Mac](AboutThisMac.png)

**Comment by stakeout55** [comment]
Yes, the 4K IPS is what I use and I believe what ZeroShadowX uses too

**Comment by ZeroShadowX** [comment]
Yes I have the same 4K Touch as well, I mentioned your repo Stakeout because Crypto mentioned he might be getting the 4K screen so just wanted to share the repo that you worked on. I know he had been battling the OLED for a while was moving on from it. My apologies if I caused any confusion.

**Comment by CryptoNeverSleeps** [comment]
No confusion... I was just trying to exhaust all efforts before making the purchase...

**Comment by stakeout55** [comment]
I understand. While I haven't seen the OLED model of this machine, I have seen them prior, and they are pretty. This IPS is actually very nice looking too so I think you'll be pleased!

**Comment by CryptoNeverSleeps** [comment]
Just purchased. Going to be a beast of a Hackintosh - 64 GB Ram

ETA Delivery - January 7th, 2021


Are you all experiencing Left Click not working on Dock... only responds to touch clicks....

**Comment by CryptoNeverSleeps** [comment]
I think we should have a moment of silence for this badass OLED screen. 60 seconds seems sufficient.

It will be missed. Perfect Blacks...  :(

**Comment by ZeroShadowX** [comment]
> Just purchased. Going to be a beast of a Hackintosh - 64 GB Ram
> 
> ETA Delivery - January 7th, 2021
> 
> Are you all experiencing Left Click not working on Dock... only responds to touch clicks....

See if this helps 
System Preferences > Trackpad 
Turn off Force Click and haptic Feedback

I've never seen the OLED but heard great things about it but I do love the 4k Touch color garmut it's beautiful 😁.

**Comment by CryptoNeverSleeps** [comment]
I just watched Gemini Man Trailer... Downloaded 1.2GB HDR .mkv also... Quality is immaculate..... going to miss this screen......

https://www.youtube.com/watch?v=vX2vsvdq8nw

**Comment by CryptoNeverSleeps** [comment]
That fixed the trackpad... Thanks... Are you both using the Fan Vent Mod?

**Comment by CryptoNeverSleeps** [comment]
I'll be making a discord for this repo... I'll send you both an invite later today.... 

Bluetooth is acting weird for my DW1820A - Wi-Fi is working fine... Bluetooth pane is now missing in settings... already cleared NVRAM....

**Comment by CryptoNeverSleeps** [comment]
Fixing Sleep - https://dortania.github.io/OpenCore-Post-Install/universal/sleep.html

I'll tackle this with OLED when I have time.

**Comment by CryptoNeverSleeps** [comment]
How are you liking the Wi-Fi card -  you linked on eBay?  Did you have to mask the pins?

Did you need any kext?

**Comment by CryptoNeverSleeps** [comment]
https://discord.gg/sNc8xK3

Discord Link

**Comment by ZeroShadowX** [comment]
> That fixed the trackpad... Thanks... Are you both using the Fan Vent Mod?

Havent heard about the Vent Mod? I'm using a DW1830 and havent run into any problems with Bluetooth using all the supplied Kext.

**Comment by CryptoNeverSleeps** [comment]
I'll keep you both updated when the NEW 4K Touchscreen IPS gets delivered...... going on within 5 minutes of delivery...  This will also give me time next year to possibly fix the OLED issue... I might buy another XPS 7590 1080P for cheap on eBay...

Join the discord channel....

**Comment by CryptoNeverSleeps** [comment]
Is your resolution with 4K Display Downsized to Look Like 1080P or the other option next to it... (Not displaying full 4K....)..

This can be seen in display settings in macOS

**Comment by CryptoNeverSleeps** [comment]
https://www.youtube.com/watch?v=uA_pOd1dMcs

MacBook Pro vs XPS 7590 4K IPS Touch vs XPS 7590 4K OLED

**Comment by stakeout55** [comment]
Haven’t looked at the fan mod. UnderVolt works phenomenally for better battery though. The card I linked of eBay requires 0 modifications of any kind. And has native support OOB.
